src/isomorphic/modern/class/__tests__/ReactES6Class-test.js

```diff
@@ -447,4 +449,62 @@ describe('ReactES6Class', () => {
     var node = ReactDOM.findDOMNode(instance);
     expect(node).toBe(container.firstChild);
   });
+
+  it('should warn when mutating the updater in the constructor', () => {
+    spyOn(console, 'error');
+
+    class Component extends React.Component {
+      constructor(props) {
+        super(props);
+        this.updater = {};
+      }
+
+      render() {
+        return null;
+      }
+    }
+
+    ReactDOM.render(<Component />, document.createElement('div'));
+
+    expect(console.error.calls.count()).toBe(1);
+    expect(console.error.calls.mostRecent().args).toMatchSnapshot();
```

Comment:
I like the instinct to use snapshots but since that's not how we test all of our other warnings, let's just use a string matcher.